NebulaServer/Server.cs

```diff
@@ -24,6 +24,11 @@ public Server()
             server = new NetManager(this)
             {
                 AutoRecycle = true,
+                // Settings below only works in DEBUG builds
+                SimulateLatency = true,
```

Comment:
Might be better to gate this regardless of whether it only applies to debug or not, as it could confuse us elsewhere.

Reply:
Yeah sure